pandas/core/series.py

```diff
@@ -4303,6 +4306,83 @@ def between(self, left, right, inclusive=True) -> "Series":
 
         return lmask & rmask
 
+    # ----------------------------------------------------------------------
+    # Convert to types that support pd.NA
+
+    def _convert_dtypes(
+        self: ABCSeries, use_nullable_dtypes: bool = True, convert_integer: bool = True
+    ) -> ABCSeries:
+        """
+        Convert objects to best possible type, and optionally,
+        columns of DataFrame or a Series to types supporting ``pd.NA``.
+
+        Parameters
+        ----------
+        use_nullable_dtypes : bool, default True
+            Whether conversion to types supporting ``pd.NA`` should be attempted.
+        convert_integer : bool, default True
+            If ``use_nullable_dtypes`` is True, Whether ``int`` types should be
+            converted to integer extension types. (Ignored if ``use_nullable_dtypes`
+            is False)
+
+        Returns
+        -------
+        Series
+            copy of Series with new (or existing) dtype
+        """
+        result = self.infer_objects()
+        if use_nullable_dtypes:
+            result = result._as_nullable_dtype(convert_integer)
+        else:
+            if is_object_dtype(result):
+                result = result.copy(deep=True)
+        return result
+
+    def _as_nullable_dtype(self: ABCSeries, convert_integer: bool = True) -> ABCSeries:
```

Comment:
i would *just* have _convert_dtypes, IOW merge these 2 functions.

Reply:
I am ok with dispatching to a helper function, but it should then live in panda.core.dtypes.cast (and not here)

Reply:
Except `pandas.core.dtypes.cast` works with arrays not Series, and I think that complicates things